roles/tendrl-ansible.tendrl-server-cleanup/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+# Task file for storage nodes
+
+- name: Stop tendrl services
+  service:
+    name: tendrl-node-agent
+    state: stopped
+
+- name: Stop etcd service
+  service:
+    name: etcd
+    state: stopped
+
+- name: Stop grafana-server service
+  service:
+    name: grafana-server
+    state: stopped
+
+- name: Stop carbon-cache service
+  service:
+    name: carbon-cache
+    state: stopped
+
+- name: Stop httpd service
+  service:
+    name: httpd
+    state: stopped
+
+- name: Cleanup etcd data
```

Comment:
isn't it destructive to remove the ETCD? User may have different namespace running for different things can we just delete the data? 

Doing it for the dev purpose makes sense to me but doing this same on production server of a client, I am a bit skeptical about it. 

thoughts? @shtripat @nthomas-redhat @anmolsachan @GowthamShanmugam

Reply:
if we are sure then the customer doesn't have anything else running on the tendrl server machine then we can take this path.

Reply:
We save tendrl data in default profile. That itself is not very correct I feel.
We can have a disclaimer in document regarding the same, that it will flush all the etcd data.

Reply:
Correct I agree shubhendu. IF some other data in default profile that will also get flushed.

Reply:
There could be another option to delete all tendrl created dirs under etcd like `/clusters`, `/nodes` and tags etc. After this keep etcd running as usual and tendrl-ansible shouldnt cause issues. We will need to test the scenario though.